Sprint_2

VagapovaLi opened the pull request:
add tests by BooksCollector class and write name test in README.md

VagapovaLi committed:
add tests by BooksCollector class and write name test in README.md

AlexKitIt commented on tests.py:
```diff
@@ -1,24 +1,123 @@
+    def test_add_new_book_success(self):
+        collector = BooksCollector()
+        collector.add_new_book('ф')
+        collector.add_new_book('Жареные зеленые помидоры в кафе Полустанок')
```
Нужно исправить: это негативная проверка, что книга не добавится, надо должна быть размещена в отдельном тесте

AlexKitIt commented on tests.py:
```diff
@@ -1,24 +1,123 @@
+
+        assert collector.books_genre['Мастер и Маргарита'] == 'Фантастика'
+
+    def test_get_book_genre_not_set(self):
```
Нужно исправить: копипаста метода test_get_book_genre

AlexKitIt commented on tests.py:
```diff
@@ -1,24 +1,123 @@
-    # напиши свои тесты ниже
-    # чтобы тесты были независимыми в каждом из них создавай отдельный экземпляр класса BooksCollector()
\ No newline at end of file
+        assert collector.favorites == expected_books_favorites
```
Нужно исправить: не проверена метод get_list_of_favorites_books

VagapovaLi committed:
Отдельно добавила негативную проверку для test_add_new_book_success
Исправила  копипаста метода test_get_book_genre
Дописала проверку для get_list_of_favorites_books

VagapovaLi committed:
Отдельно добавила негативную проверку для test_add_new_book_success
Исправила  копипаста метода test_get_book_genre
Дописала проверку для get_list_of_favorites_books

VagapovaLi committed:
Отдельно добавила негативную проверку для test_add_new_book_success
Исправила  копипаста метода test_get_book_genre
Дописала проверку для get_list_of_favorites_books

VagapovaLi committed:
добавила вызов метода def get_list_of_favorites_books
